composer.json

```diff
@@ -27,7 +27,7 @@
         "symfony/form": "^2.7 || ^3.0 || ^4.0",
         "symfony/yaml": "^2.7 || ^3.0 || ^4.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0 || ^4.0",
-        "phpunit/phpunit": "^5.6"
+        "phpunit/phpunit": "^5.7"
```

Comment:
As in many other repos, this should be `^5.7|^6.4`.

Reply:
Gonna work on that!

Reply:
@alcaeus Please merge this namespaces changes now. I'll work on PHPUnit 6 support latter, a lot of stuff broken.